RecoTracker/LSTCore/src/alpaka/NeuralNetwork.h

```diff
@@ -239,6 +240,107 @@ namespace ALPAKA_ACCELERATOR_NAMESPACE::lst {
     }
   }  // namespace t5dnn
 
+  namespace t5embdnn {
+    template <typename TAcc>
+    ALPAKA_FN_ACC ALPAKA_FN_INLINE void runEmbed(
+        TAcc const&        acc,
+        MiniDoubletsConst  mds,
+        unsigned int       mdIndex1,
+        unsigned int       mdIndex2,
+        unsigned int       mdIndex3,
+        unsigned int       mdIndex4,
+        unsigned int       mdIndex5,
+        float              innerRadius,
+        float              outerRadius,
+        float              bridgeRadius,
+        float (&embedding)[6]
+    ) {
+      constexpr unsigned int kinputFeatures = 23;
+      constexpr unsigned int khidden        = 32;
+      constexpr unsigned int kembed         = 6;
+
+      float eta1 = alpaka::math::abs(acc, mds.anchorEta()[mdIndex1]);
+      float eta2 = alpaka::math::abs(acc, mds.anchorEta()[mdIndex2]);
+      float eta3 = alpaka::math::abs(acc, mds.anchorEta()[mdIndex3]);
+      float eta4 = alpaka::math::abs(acc, mds.anchorEta()[mdIndex4]);
+      float eta5 = alpaka::math::abs(acc, mds.anchorEta()[mdIndex5]);
+
+      float phi1 = mds.anchorPhi()[mdIndex1];
+      float phi2 = mds.anchorPhi()[mdIndex2];
+      float phi3 = mds.anchorPhi()[mdIndex3];
+      float phi4 = mds.anchorPhi()[mdIndex4];
+      float phi5 = mds.anchorPhi()[mdIndex5];
+
+      float z1 = alpaka::math::abs(acc, mds.anchorZ()[mdIndex1]);
+      float z2 = alpaka::math::abs(acc, mds.anchorZ()[mdIndex2]);
+      float z3 = alpaka::math::abs(acc, mds.anchorZ()[mdIndex3]);
+      float z4 = alpaka::math::abs(acc, mds.anchorZ()[mdIndex4]);
+      float z5 = alpaka::math::abs(acc, mds.anchorZ()[mdIndex5]);
+
+      float r1 = mds.anchorRt()[mdIndex1];
+      float r2 = mds.anchorRt()[mdIndex2];
+      float r3 = mds.anchorRt()[mdIndex3];
+      float r4 = mds.anchorRt()[mdIndex4];
+      float r5 = mds.anchorRt()[mdIndex5];
+
+      float x[kinputFeatures] = {
+          eta1 / dnn::kEta_norm,
+          alpaka::math::abs(acc, phi1) / dnn::kPhi_norm,
+          z1 / dnn::t5dnn::kZ_max,
+          r1 / dnn::t5dnn::kR_max,
+
+          eta2 - eta1,
+          cms::alpakatools::deltaPhi(acc, phi2, phi1) / dnn::kPhi_norm,
+          (z2 - z1) / dnn::t5dnn::kZ_max,
+          (r2 - r1) / dnn::t5dnn::kR_max,
+
+          eta3 - eta2,
+          cms::alpakatools::deltaPhi(acc, phi3, phi2) / dnn::kPhi_norm,
+          (z3 - z2) / dnn::t5dnn::kZ_max,
+          (r3 - r2) / dnn::t5dnn::kR_max,
+
+          eta4 - eta3,
+          cms::alpakatools::deltaPhi(acc, phi4, phi3) / dnn::kPhi_norm,
+          (z4 - z3) / dnn::t5dnn::kZ_max,
+          (r4 - r3) / dnn::t5dnn::kR_max,
+
+          eta5 - eta4,
+          cms::alpakatools::deltaPhi(acc, phi5, phi4) / dnn::kPhi_norm,
+          (z5 - z4) / dnn::t5dnn::kZ_max,
+          (r5 - r4) / dnn::t5dnn::kR_max,
+
+          alpaka::math::log10(acc, innerRadius),
```

Comment:
from the resolution point of view R of 1. m vs 1.2 m are very different, while 1e3 m and 1e4 m are about the same.
At some point I thought that taking a `log10` regularizes the dynamic range, but even in this example it's clearly not the case for a simple linear transformation from `log10(r)`.
Does the network learn a related transformation? (from the depth used, I'm guessing it's not really the case).

Would a more common use of `1/radius` (perhaps in meters, if it also takes an extra cost to normalize) be more performant?